include/ginkgo/config.hpp.in

```diff
@@ -108,7 +108,7 @@
 
 /* Do we need to use blocking communication in our SpMV? */
 // clang-format off
-#cmakedefine GINKGO_FORCE_SPMV_BLOCKING_COMM
+#cmakedefine01 GINKGO_HAVE_OPENMPI_PRE_4_1_X
```

Comment:
I can add the old macro again, but I'm not really sure how we usually consider macros for interface stability. IMO we either should state which macros are public, and which are private. Because this macro should definitely be private.

Reply:
some of them I might consider it is public like version and propabably mixed precision.
This is only used in the experimental feature, so I also think it is free to change